wren-ui/src/apollo/server/dataSource.ts

```diff
@@ -151,6 +152,22 @@ const dataSource = {
     CLICK_HOUSE_CONNECTION_INFO,
     UrlBasedConnectionInfo
   >,
+  [DataSourceName.TRINO]: {
+    sensitiveProps: ['password'],
+    toIbisConnectionInfo(connectionInfo) {
+      const { catalog, host, password, port, schema, ssl, username } =
+        decryptConnectionInfo(
+          DataSourceName.TRINO,
+          connectionInfo,
+        ) as TRINO_CONNECTION_INFO;
+      console.log('connectionInfo', connectionInfo);
```

Comment:
I guess you forgot to remove the console =D

Reply:
Will remove that